apps/charterafrica/src/lib/tools/models.js

```diff
@@ -0,0 +1,132 @@
+import { FetchError } from "../../utils/fetchJson";
+import api from "../payload";
+
+export const ORGANIZATION_COLLECTION = "organisations";
+export const PEOPLE_COLLECTION = "people";
+export const TOOL_COLLECTION = "tools";
+export const DIGITAL_DEMOCRACY_ECOSYSTEM = "digital-democracy-ecosystem-config";
+
+function removeDuplicatesByKey(arr, key) {
+  const uniqueKeys = new Set();
+  return (arr || []).filter((obj) => {
+    if (!obj) {
+      return false;
+    }
+    const objKey = obj[key];
+    if (!uniqueKeys.has(objKey)) {
+      uniqueKeys.add(objKey);
+      return true;
+    }
+    return false;
+  });
+}
+
+export const createOrganization = async (toCreate) => {
+  const { docs } = await api.getCollection(ORGANIZATION_COLLECTION, {
+    where: {
+      externalId: { equals: toCreate.externalId },
+    },
+  });
+  if (docs.length) {
+    const data = await api.updateCollection(
+      ORGANIZATION_COLLECTION,
+      docs[0]?.id,
+      {
+        ...toCreate,
+        updatedAt: new Date(),
+      }
+    );
+    return data;
+  }
+  const data = await api.createCollection(ORGANIZATION_COLLECTION, toCreate);
+  return data;
+};
+
+export const createPerson = async (toCreate) => {
+  const { docs } = await api.getCollection(PEOPLE_COLLECTION, {
+    where: {
+      externalId: { equals: toCreate.externalId },
+    },
+  });
+  if (docs.length) {
+    const data = await api.updateCollection(PEOPLE_COLLECTION, docs[0]?.id, {
+      ...toCreate,
+      updatedAt: new Date(),
+    });
+    return data;
+  }
+  const data = await api.createCollection(PEOPLE_COLLECTION, toCreate);
+  return data;
+};
+
+export const bulkCreatePeople = async (contributors = []) => {
+  const promises = contributors?.map(async (contributor) => {
```

Comment:
```suggestion
export const bulkCreatePeople = async (contributors = []) => {
  const promises = contributors.map(async (contributor) => {
```
You're already setting `[]` as default, no?